sdks/java/extensions/sql/zetasql/build.gradle

```diff
@@ -31,25 +32,23 @@ def zetasql_version = "2020.10.1"
 
 dependencies {
   compile enforcedPlatform(library.java.google_cloud_platform_libraries_bom)
-  compile project(":sdks:java:core")
+  compile project(path: ":sdks:java:core", configuration: "shadow")
   compile project(":sdks:java:extensions:sql")
   compile library.java.vendored_calcite_1_20_0
   compile library.java.guava
   compile library.java.grpc_api
   compile library.java.protobuf_java
-  compile library.java.protobuf_java_util
-  compile "com.google.api.grpc:proto-google-common-protos:1.12.0" // Interfaces with ZetaSQL use this
-  compile "com.google.api.grpc:grpc-google-common-protos:1.12.0" // Interfaces with ZetaSQL use this
-  compile "com.google.zetasql:zetasql-jni-channel:$zetasql_version"
   compile "com.google.zetasql:zetasql-client:$zetasql_version"
   compile "com.google.zetasql:zetasql-types:$zetasql_version"
+  compile library.java.joda_time
+  compile library.java.vendored_guava_26_0_jre
+  permitUnusedDeclared library.java.protobuf_java_util
```

Comment:
Since you removed library.java.protobuf_java_util, you shouldn't need to `permitUnusedDeclared` any more.